lib/rutema/core/framework.rb

```diff
@@ -83,30 +83,33 @@ class RunnerMessage<Message
     # * "status" - the status of the respective step
     # * +:timestamp+ - most often the timestamp of the creation of the message,
     #   defaults to +Time.now+
-
     def initialize params
       super(params)
       @duration=params.fetch("duration",0)
       @status=params.fetch("status",:none)
       @number=params.fetch("number",1)
       @out=params.fetch("out","")
       @err=params.fetch("err","")
+      @backtrace=params.fetch("backtrace","")
+      @is_special=params.fetch("is_special","")
     end
 
     ##
     # Convert the instance to a convenient textual representation
     def to_s
       msg="#{@test}:"
+      msg<<" #{@timestamp.strftime("%H:%M:%S")} :"
       msg<<"#{@text}." unless @text.empty?
       outpt=output()
-      msg<<" Output:\n#{outpt}" unless outpt.empty? || @status!=:error
+      msg<<" Output" + (outpt.empty? ? "." : ":\n#{outpt}") # unless outpt.empty? || @status!=:error
```

Comment:
Why the comment on the right? Should we just delete that?

Reply:
Good point :+1: Still I'd leave it for the full refactoring and drop it then, when I go through all outdated and/or superfluous comments. Maybe this is a valuable hint when taking a deeper look at the code.